src/test/a11y/mocks.ts

```diff
@@ -1,197 +1,33 @@
-import * as moment from 'moment'
-import { LocalDate } from 'app/forms/models/localDate'
-import { MobilePhone } from 'app/forms/models/mobilePhone'
-import InterestDate from 'app/claims/models/interestDate'
-import InterestDateType from 'app/common/interestDateType'
-import Interest, { InterestType } from 'app/forms/models/interest'
-import { Defendant, Defendant as DraftDefendant } from 'app/drafts/models/defendant'
-import { TheirDetails } from 'app/claims/models/details/theirs/theirDetails'
-import { Company as CompanyDetails } from 'app/claims/models/details/theirs/company'
-import { Individual } from 'app/claims/models/details/yours/individual'
-import { default as DraftClaimant } from 'app/drafts/models/claimant'
-import { IndividualDetails } from 'forms/models/individualDetails'
-import { Address } from 'claims/models/address'
-import { RangeGroup } from 'fees/models/rangeGroup'
-import { DraftCCJ } from 'ccj/draft/draftCCJ'
-import { PaidAmount } from 'ccj/form/models/paidAmount'
-import { PaidAmountOption } from 'ccj/form/models/yesNoOption'
-import { PartyDetails } from 'forms/models/partyDetails'
-import { CountyCourtJudgment } from 'claims/models/countyCourtJudgment'
-import { PaymentType } from 'ccj/form/models/ccjPaymentOption'
-import { Draft } from 'models/draft'
-import { DraftDocument } from 'models/draftDocument'
 
-function mockedClaimDraft () {
-  let draft = new DraftClaim()
-  draft.readResolveDispute = true
-  draft.claimant = new DraftClaimant()
-  let individualClaimant: IndividualDetails = draft.claimant.partyDetails = new IndividualDetails()
-  individualClaimant.dateOfBirth = new DateOfBirth(true, new LocalDate(1980, 1, 1))
-  individualClaimant.name = 'John Smith'
-  draft.claimant.partyDetails = new PartyDetails()
-  draft.claimant.partyDetails.address.postcode = 'postcode'
-  draft.claimant.partyDetails.address.line1 = 'line1'
-  draft.claimant.mobilePhone = new MobilePhone('07873738765')
+import * as idamServiceMock from '../http-mocks/idam'
+import * as draftStoreMock from '../http-mocks/draft-store'
+import * as claimStoreMock from '../http-mocks/claim-store'
+import * as feesMock from '../http-mocks/fees'
 
-  draft.defendant = new DraftDefendant()
-  let individualDefendant: IndividualDetails = draft.defendant.partyDetails = new IndividualDetails()
-  draft.defendant.email = new Email('defendant@hmcts.com')
-  individualDefendant.address.postcode = 'postcode'
-  individualDefendant.address.line1 = 'line1'
-  individualDefendant.dateOfBirth = new DateOfBirth(true, new LocalDate(1980, 1, 1))
-  draft.defendant.mobilePhone = new MobilePhone('07873738765')
-  draft.defendant.partyDetails = new PartyDetails()
-  draft.defendant.partyDetails.address.postcode = 'postcode'
-  draft.defendant.partyDetails.address.line1 = 'line1'
-  draft.interestDate.date = new LocalDate(2017, 7, 21)
+idamServiceMock.resolveRetrieveUserFor(1, 'cmc-private-beta', 'letter-holder', 'claimant', 'defendant').persist()
```

Comment:
A `persist()` method keeps mock in place forever which is different to the ordinary behaviour where mock is being removed after being used for the first time.